server/src/main/java/org/apache/iotdb/db/engine/storagegroup/StorageGroupProcessor.java

```diff
@@ -86,23 +86,14 @@
 
 /**
  * For sequence data, a StorageGroupProcessor has some TsFileProcessors, in which there is only one
- * TsFileProcessor in the working status. <br/>
- * <p>
- * There are two situations to set the working TsFileProcessor to closing status:<br/>
- * <p>
- * (1) when inserting data into the TsFileProcessor, and the TsFileProcessor shouldFlush() (or
- * shouldClose())<br/>
- * <p>
- * (2) someone calls waitForAllCurrentTsFileProcessorsClosed(). (up to now, only flush command from
- * cli will call this method)<br/>
- * <p>
- * UnSequence data has the similar process as above.
- * <p>
- * When a sequence TsFileProcessor is submitted to be flushed, the updateLatestFlushTimeCallback()
- * method will be called as a callback.<br/>
- * <p>
- * When a TsFileProcessor is closed, the closeUnsealedTsFileProcessor() method will be called as a
- * callback.
+ * TsFileProcessor in the working status. <br/> <p> There are two situations to set the working
+ * TsFileProcessor to closing status:<br/> <p> (1) when inserting data into the TsFileProcessor, and
+ * the TsFileProcessor shouldFlush() (or shouldClose())<br/> <p> (2) someone calls
+ * waitForAllCurrentTsFileProcessorsClosed(). (up to now, only flush command from cli will call this
+ * method)<br/> <p> UnSequence data has the similar process as above. <p> When a sequence
+ * TsFileProcessor is submitted to be flushed, the updateLatestFlushTimeCallback() method will be
+ * called as a callback.<br/> <p> When a TsFileProcessor is closed, the
+ * closeUnsealedTsFileProcessor() method will be called as a callback.
```

Comment:
Hi, do not FORMAT this paragraph...

Reply:
Sorry my fault... I have reversed all formats like this.